Mindscape.Raygun4Net.NetCore.Common/FireAndForget.cs

```
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Mindscape.Raygun4Net
+{
+  internal class Fire
+  {
+
+    /// <summary>
+    /// If you need to see the details of an exception thrown please, add a logging action here.
+    /// </summary>
+    internal static Action<string> DebugLog { get; set; } = message => { /* NOOP by default */};
```

Is this property meant to be something that is user configurable? I note that it's not settable in the public api surface, and we don't provide a way to override it?

---

Currently we have a if (_settings.ThrowOnError) for debugging.

However that is pretty heavy handed and would not work where this is used because Fire.AndForget explicitly swallows any errors.

I have ideas for making a  public debug log (even if we tag it  that we make no guaranteed to support it etc)
Similar to what we have in raygun4js.


I would also like to have some on onboarding setup debug helper (maintained API) (e.g., you apikey is empty, invalid, there is no network connection etc)


That is all future direction stuff.
This is just enough for us to have a way of debugging